packages/template-retail-react-app/app/components/_app-config/index.test.js

```diff
@@ -47,6 +57,13 @@ describe('AppConfig', () => {
             </StaticRouter>
         )
         expect(container).toBeDefined()
+
+        // Wait for access token to be saved
+        // Otherwise, the test would end prematurely before our component has finished its business
+        // (for example: commerce-sdk-react Provider needs to finish its useEffect for `auth.ready()`)
+        await waitFor(() => {
+            expect(window.localStorage.setItem).toHaveBeenCalled()
```

Comment:
Finally this AppConfig test is fixed 😌 Previously it failed every time on my machine, and it was failing intermittently on CI.